dhall-docs/src/Dhall/data/index.css

```diff
@@ -105,6 +105,62 @@
     unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
 
-h1, p {
+/*
+
+primary-color: #C4C4C4
```

Comment:
It might be worth using .sass files to define variables and avoid repeating the same color values on several places. There is a haskell package that produces css from [sass], [hsass] but doesn't look very promising. Another solution could be to add some kind of pre-processing using `node` but it might complicate things.

[sass]: https://sass-lang.com/
[hsass]: https://hackage.haskell.org/package/hsass